frontend/cypress/integration/updateProject.spec.js

```diff
@@ -1,136 +1,5 @@
-    cy.wait('@sign_in')
-
-    cy.intercept(`http://gitea.kitspace.test:3000/api/v1/repos/**`).as('getRepo')
-    cy.visit(updatePageRoute)
-  })
-
-  it('should render the update page with correct project name', () => {
-    cy.visit(updatePageRoute)
-    cy.get('[data-cy=update-form-name] > input').should('have.value', testRepoName)
-  })
-
-  it('should handle uploading files', () => {
-    cy.visit(updatePageRoute)
-    // Intercept request for the upload route
-    cy.intercept(
-      `http://gitea.kitspace.test:3000/${testRepoFullName}/upload-file**`,
-    ).as('upload')
-    // Intercept request for the commit route
-    cy.intercept(
-      `http://gitea.kitspace.test:3000/${testRepoFullName}/upload/master**`,
-    ).as('commit')
-
-    cy.preFileDrop(username)
-    cy.fixture('example2.png', 'base64').then(file => {
-      cy.get('.dropzone').dropFiles([file], ['example2.png'])
-    })
-
-    // waiting prevents random test failures due to jittering in response
-    cy.wait('@upload')
-    // Dropping a file should make it appear in the preview component
-    cy.get('[data-cy=file-name]').contains('example2.png')
-
-    // Commit files to the repo
-    cy.get('[data-cy=update-form-submit]').click()
-    cy.wait('@commit')
-
-    // After reloading the update page the files should still in the preview component
-    cy.reload()
-    cy.get('[data-cy=file-name]').contains('example2.png')
-  })
-
-  it('should handle changing project name', () => {
-    const newName = 'new-cool-name'
-
-    // Changing the project name a submitting it
-    cy.get('[data-cy=update-form-name] > input').clear().type(newName)
-    cy.get('[data-cy=update-form-submit]').click()
-    cy.wait('@getRepo')
-
-    // should redirect to the new update page
-    cy.url().should('contain', `/${username}/${newName}`)
-  })
-
-  it('should handle updating project name and uploading files at the same time', () => {
-    // notice that the project name has changed to `new-cool-name` in the previous test
-    const testRepoFullName = `${username}/new-cool-name`
-    const newName = 'even-cooler-name'
-
-    cy.visit(`/${testRepoFullName}`)
-
-    // Update project name to `even-cooler-name`
-    cy.get('[data-cy=update-form-name] > input').clear().type(newName)
-
-    // Intercept request for the uploading route
-    cy.intercept(
-      `http://gitea.kitspace.test:3000/${testRepoFullName}/upload-file**`,
-    ).as('upload')
-
-    // Upload `example3.png`
-    cy.preFileDrop(username)
-    cy.fixture('example3.png', 'base64').then(file => {
-      cy.get('.dropzone').dropFiles([file], ['example3.png'])
-    })
-
-    cy.wait('@upload')
-    // Submit the update form
-    cy.get('[data-cy=update-form-submit]').click()
-
-    // Should redirect to the new update page
-    cy.url().should('contain', `/${username}/${newName}`)
-
-    // `example3.png` should be in the preview
-    cy.get('[data-cy=file-name]').contains('example3.png')
-  })
-})
-
```

Comment:
How come this whole test is being deleted?

Reply:
This is a mistake! I looked at the history for this file, I deleted it in an unrelated commit to this file, will roll that part back.